frontend/src/scenes/persons/PersonPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import { useValues } from 'kea'
+import { personLogic } from './personLogic'
+import { Spinner } from 'lib/lemon-ui/Spinner'
+import { ProfilePicture } from 'lib/lemon-ui/ProfilePicture'
+import { LemonButton, Link } from '@posthog/lemon-ui'
+import { urls } from 'scenes/urls'
+import { PropertiesTable } from 'lib/components/PropertiesTable'
+import { PropertyDefinitionType } from '~/types'
+import { IconOpenInNew } from 'lib/lemon-ui/icons'
+import { asDisplay } from './person-utils'
+
+export type PersonPreviewProps = {
+    distinctId: string | undefined
+}
+
+export function PersonPreview(props: PersonPreviewProps): JSX.Element {
+    if (!props.distinctId) {
+        return <></>
```

Comment:
Empty fragments are not good for representing empty results, e.g. they're not falsy. `null` is better